torchci/rockset/commons/__sql/test_time_per_oncall.sql

```diff
@@ -0,0 +1,15 @@
+SELECT
```

Comment:
These two queries looks very much similar.  I suspect that you could try to combine them into one:

* If `oncall` param is set, returns the logic in `test_time_per_oncall`
* If `oncall` param is an empty string, returns pretty much the same thing but with `oncall` set to an empty string and data aggregated accordingly

Reply:
Using `WITH a_new_temp_table AS (SELECT ...) SELECT * FROM a_new_temp_table` would be an useful pattern to use here I think

Reply:
The difference is a bit subtle unfortunately. Some tests have multiple oncalls, so it may be better to keep them two separate queries to avoid overcounting.